src/c++/vdo/base/vio.c

```diff
@@ -128,7 +128,9 @@ int create_multi_block_metadata_vio(struct vdo *vdo, enum vio_type vio_type,
 	struct vio *vio;
 	int result;
 
-	/* If struct vio grows past 256 bytes, we'll lose benefits of VDOSTORY-176. */
+	/* If struct vio grows past 256 bytes, we'll lose the benefits of having 50% memory
+	 * overhead.
+	 */
 	BUILD_BUG_ON(sizeof(struct vio) > 256);
 
```

Comment:
I had to look up what this story was about. It was actually due to a complaint that if we set the cache size to 40G, VDO would allocate something closer to 60G instead. Ultimately it turned out the vio structures were way bigger than they needed to be, especially for metadata vios.

So I think this entire comment can get removed; I don't think there's much we can say about wanting to keep struct vio small.

Reply:
Okay - yeah, this one confused me too. Wasn't clear what the benefits of the referenced ticket would be.